README.md

```diff
@@ -1,2 +1,75 @@
 # linux-workstation-data-engineering
-Collection of Ansible roles related to Data Engineering technologies to be run against UKHO/linux-workstation
+
+This is to be run on-top of the linux-workstation provided through `UKHO/linux-workstation`
+
+To run the playbook, run the following in the base of the repository:
+
+```
+$ sudo ansible-playbook ansible/playbook.yml -i ansible/inventories/laptop-vm/hosts --extra-vars="user_home=$HOME"
```

Comment:
sudo not required. When running from a shared folder, running as sudo gives the error 'playbook.yml not found'. Haven't discovered why this is, but running without sudo works OK.

Reply:
Leaving as is, as discussed